software/control/gui_hcs.py

```diff
@@ -264,6 +273,9 @@ def setupHardware(self):
 
         self.microcontroller.reset()
         time.sleep(0.5)
+        if USE_SQUID_FILTERWHEEL is True:
```

Comment:
You can just do `if USE_SQUID_FILTERWHEEL:` like you did above.  The `is True` isn't needed.  Same for all the ones below.

Reply:
@veerwang

Reply:
Sure